package.json

```diff
@@ -45,7 +46,7 @@
     "release:standard-version": "standard-version",
     "release:changelog": "echo \"$(head -3 CHANGELOG.md)\n\n$(node_modules/.bin/conventional-changelog -p angular)\n\n$(tail -n +4 CHANGELOG.md)\" > CHANGELOG.md",
     "release": "npm run release:standard-version && npm run release:changelog",
-    "prepare": "husky install"
+    "prepare": "husky || true"
```

Comment:
Can you explain the purpose behind this change?

Reply:
The install command is deprecated. Husky recommends adding the `|| true` so the command will never fail